### ISSUE TITLE
"staging server's URL is"

### ISSUE THREAD

**Issue by mjordan** [ubermichael/westvault-vagrant#4]
Logging into the staging server presents this message to the user:

The staging server's URL is http://localhost:8181/westvaultpln/api/sword/2.0/sd-iri

Might be useful to clarify that this is the staging server's SWORD URL or something similar. Not a big deal, but a user might wonder why they are being told this since they presumably are at the staging server's URL already.

**Comment by ubermichael** [comment]
Good point. I kept forgetting the URL when I was configuring OC during dev and instead of having to look it up every time I just added it to the home page. Lazy, yes. But also very lazy.
